src/sentry/snuba/query_subscription_consumer.py

```diff
@@ -87,9 +87,14 @@ def parse_message_value(value: bytes, jsoncodec: JsonCodec) -> PayloadV3:
     # break things. This should convert the payload into a class rather than passing
     # the dict around, but until we get time to refactor we can keep things working
     # here.
-    payload.setdefault("values", payload.get("result"))  # type: ignore
-    payload["timestamp"] = parse_date(payload["timestamp"]).replace(tzinfo=pytz.utc)  # type: ignore
-    return payload
+    return {
+        "entity": payload["entity"],
+        "subscription_id": payload["subscription_id"],
+        "request": payload["request"],
+        "result": payload["result"],
```

Comment:
That's what the test expects, do you think the test is wrong?

Reply:
Yes

Reply:
But what I don't get is why the typing is not failing since these keys don't seem to be on SubscriptionUpdate

Reply:
I think you have moved the SubscriptionUpdate into a file that is now not included in mypy.ini so the typechecker isn't telling you about the error

Reply:
I don't think we need `result` in the `SubscriptionUpdate`. Basically this is a side effect of us using the `payload` dict directly, rather than tranforming it into a dataclass that we pass around. 

In the v1 message, `result` was called `values`, and then when we renamed it in v2 we just copied `result` over into the `values` field of the payload so we didn't break anything. But we also left it in place in the dict, even though it was unused.

I think we're fine to remove it here and from the test that is expecting it.

Reply:
I'll have to make a change to the schema again to not expect `result`.

Reply:
Isn't the schema still passing result though? We just want it in the `values` key

Reply:
Please don't change the schema, just remove these two lines with "request" and "result"